packages/localize/stories/10-overview.stories.mdx

```diff
@@ -0,0 +1,119 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { render } from '@lion/core';
+import { localize, formatNumber, formatDate } from '../index.js'
+
+<Meta title="Localize System/Localize" />
+
+<Story name="Default"></Story>
```

Comment:
Empty story

Reply:
right now needed to show a menu point 🙈 
don't know if there is another way... gonna ask in storybook (and give this as an example)
e.g. is a workaround to get the stories.js <> stories.mdx somewhat working